app/assets/stylesheets/components/_super-navigation-header.scss

```diff
@@ -0,0 +1,90 @@
+$breakpoint: 769px;
+
+// Remove bottom borders
+.govuk-header__super-navigation,
+.govuk-header__super-navigation-container {
+  border-bottom: 0;
+  margin-bottom: 0;
+  padding-top: 0;
+}
+
+// logo
+.govuk-header__super-navigation-header-logo {
+  padding-top: govuk-spacing(2) + 4;
+  padding-bottom: govuk-spacing(2);
+  margin-bottom: 0;
+}
+
+// header content
+.govuk-header__super-navigation-content {
+  @include govuk-media-query($from: $breakpoint) {
+    text-align: right;
+  }
+}
+
+.govuk-header__super-navigation-items {
+  margin-bottom: 0;
+}
+
+.govuk-header__super-navigation-item {
+  display: flex;
+  margin-right: 0;
+  padding: 0;
+
+  @include govuk-media-query($from: $breakpoint) {
+    display: inline-flex;
+  }
+}
+
+.govuk-header__super-navigation-item-link {
+  padding: govuk-spacing(3) + 2;
+  width: 100%;
+
+  @include govuk-media-query($from: $breakpoint) {
+    width: auto;
+  }
+}
+
+// Search
+
+.govuk-header__super-navigation-item--search {
+  @include govuk-media-query($from: $breakpoint) {
+    height: 55px;
+    width: 55px;
+    padding: 0;
+    margin-right: govuk-spacing(3);
+    background-color: govuk-colour('blue');
+  }
+}
+
+.govuk-header__super-navigation-item-link--search {
+  @include govuk-media-query($from: $breakpoint) {
+    background-image: image-url("search-icon.svg");
+    background-position: 50%;
+    background-size: 35%;
+    background-repeat: no-repeat;
+    display: inherit;
+    width: 55px;
+
+    &:focus {
+      background-image: image-url("search-icon--black.svg");
+    }
+  }
+}
+
+.govuk-header__super-navigation-item-link-text--search {
+  @include govuk-media-query($from: $breakpoint) {
+    display: block;
+    text-indent: -5000px;
+    overflow: hidden;
+  }
+}
```

Comment:
I think that this would be better off using either the mixin or the styles from [`govuk-visually-hidden`](https://github.com/alphagov/govuk-frontend/blob/4b5ed0ada3b64722ae2074dfa0db5d4d3a45cb79/src/govuk/helpers/_visually-hidden.scss#L16) as it's a better way of making things available to screenreaders.

Reply:
Tried that - didn't work because it has absolute positioning